src/api/passport.js

```diff
@@ -0,0 +1,23 @@
+import bcrypt from 'bcryptjs';
+
+export const validatePassword = function (passport, password, next) {
+  bcrypt.compare(password, passport.password, next);
+};
+
+export async function hashPassword(password) {
```

Comment:
Shouldn't this be a helper function ? we can move it to "utils" folder  ?

Reply:
yes sure :)